mkt/site/middleware.py

```diff
@@ -111,7 +111,7 @@ def process_request(self, request):
             if stored_ov_lang.lower() in settings.LANGUAGE_URL_MAP:
                 ov_lang = stored_ov_lang
 
-        if 'lang' in request.REQUEST:
+        if 'lang' in request.GET:
```

Comment:
I can convert this easily enough but can we find where we might be doing this? Would it be fireplace?